development/vim/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Install vim
```

Comment:
We should decide on one way to structure the tasks. Inside a `block` like above or without one. I suggest the former.